eda_analysis/eda_analysis.py

```diff
@@ -145,4 +149,47 @@ def calc_cor(dataframe, num_vars):
     >>>num_var = ['height', 'width']
     >>> calc_cor(X, num_vars)    
     """
-    # Code
\ No newline at end of file
+    # Test input 'dataframe' is a dataframe
+    assert isinstance(dataframe, pd.DataFrame), "Input 'dataframe' is not a dataframe"
+    
+    df_num = dataframe.loc[:, num_var]
+    df_num = df_num.dropna()
+    
+    for i in num_var:
+        assert np.issubdtype(dataframe[i].dtype, np.number), "Columns are not all numeric"
```

Comment:
I think in Python, this part should be done using `if` and `raise`. `assert` is used in test function.

Reply:
And I think it's better to add more `if` to catch other wrong input like `num_var` contains the column names not in the dataframe and `num_var` should be a list of strings as well.

Reply:
Changed in the new commit and added tests like the describe_num_var.